README.md

```
@@ -1,2 +1,126 @@
+# Skyline
+
+[![PyPI version](https://badge.fury.io/py/partcad-skyline.svg)](https://pypi.org/project/partcad-skyline/)
+
+## What is Skyline?
+
+Skyline is a CLI tool that generates 3D visualizations of your development activity, creating a virtual cityscape where
+each building represents your contributions.
+
+> ## The City of One
+>
+> <img align="right" src="./docs/assets/images/skyline.png" alt="Description" width="350"/>
```

Markdownlint (0.35.0)

Element: img
Inline HTML

(MD033, no-inline-html)